src/ngraph/runtime/gpu/cuda_emitter.hpp

```diff
@@ -174,7 +174,7 @@ namespace ngraph
                                          NVShape input_dilation,
                                          NVDiff input_pad_below);
 
-                size_t build_concat(const std::vector<std::string>& dtypes,
+                size_t build_concat(const std::string& dtypes,
```

Comment:
dtypes->dtype for consistency w/ implementation

Reply:
changed.